packages/features/bookings/lib/handleNewBooking/test/workflow-notifications.test.ts

```diff
@@ -0,0 +1,541 @@
+              externalId: "other-team-member-1@google-calendar.com",
+            },
+          },
+        ];
+
+        await createBookingScenario(
+          getScenarioData({
+            workflows: [
+              {
+                teamId: 1,
+                trigger: "NEW_EVENT",
+                action: "EMAIL_HOST",
+                template: "REMINDER",
+                activeEventTypeId: 1,
+              },
+              {
+                teamId: 1,
+                trigger: "NEW_EVENT",
+                action: "SMS_ATTENDEE",
+                template: "REMINDER",
+                activeEventTypeId: 1,
+              },
+            ],
+            eventTypes: [
+              {
+                id: 1,
+                slotInterval: 15,
+                schedulingType: SchedulingType.COLLECTIVE,
+                length: 15,
+                users: [
+                  {
+                    id: 101,
+                  },
+                  {
+                    id: 102,
+                  },
+                ],
+                teamId: 1,
+                destinationCalendar: {
+                  integration: TestData.apps["google-calendar"].type,
+                  externalId: "event-type-1@google-calendar.com",
+                },
+              },
+            ],
+            organizer,
+            usersApartFromOrganizer: otherTeamMembers,
+            apps: [TestData.apps["google-calendar"], TestData.apps["daily-video"]],
+          })
+        );
+
+        mockSuccessfulVideoMeetingCreation({
+          metadataLookupKey: "dailyvideo",
+          videoMeetingData: {
+            id: "MOCK_ID",
+            password: "MOCK_PASS",
+            url: `http://mock-dailyvideo.example.com/meeting-1`,
+          },
+        });
+
+        mockCalendarToHaveNoBusySlots("googlecalendar", {
+          create: {
+            id: "MOCKED_GOOGLE_CALENDAR_EVENT_ID",
+            iCalUID: "MOCKED_GOOGLE_CALENDAR_ICS_ID",
+          },
+        });
+
+        const mockBookingData = getMockRequestDataForBooking({
+          data: {
+            start: `${getDate({ dateIncrement: 1 }).dateString}T09:00:00.000Z`,
+            end: `${getDate({ dateIncrement: 1 }).dateString}T09:15:00.000Z`,
+            user: organizer.username,
+            eventTypeId: 1,
+            responses: {
+              email: booker.email,
+              name: booker.name,
+              location: { optionValue: "", value: BookingLocations.CalVideo },
+              smsReminderNumber: "000",
+            },
+          },
+        });
+
+        const { req } = createMockNextJsRequest({
+          method: "POST",
+          body: mockBookingData,
+        });
+
+        await handleNewBooking(req);
+
+        expectSMSWorkflowToBeTriggered({
+          sms,
+          toNumber: "000",
+        });
+
+        expectWorkflowToBeTriggered({
+          // emailsToReceive: [organizer.email].concat(otherTeamMembers.map(member => member.email)),
```

Comment:
We currently only send to one of the hosts for team events. This needs to be fixed. I created an issue for that: https://github.com/calcom/cal.com/issues/14739